### ISSUE TITLE
Unicode issue

### ISSUE THREAD

**Issue by THEtomaso** [billthefarmer/editor#128]
Not exactly sure how this works in Android, but every single text editor that I've tried (Including Editor) seems to be unable to display unicode characters.
Don't know if it's a system-related issue, or if the apps themselves simply lacks unicode support.

Example:
**é** is displayed as **�**

Any way to mend this in Editor?

**Comment by billthefarmer** [comment]
Are you sure it's unicode? I have no difficulty entering unicode characters, which are displayed correctly. I also had a look with a hex editor, they look like unicode to me. Notepad says they are UTF-8. And they work on here: èéēêëõôæ.

![Screenshot_20210515-161208 1](https://user-images.githubusercontent.com/3448386/118373620-cd1bfd80-b5af-11eb-9aad-35363752afdb.png)

**Comment by THEtomaso** [comment]
Thanks for your quick reply.
..and you're right.
The .txt files that I were opening seems to have been saved using ANSI coding, rather than UTF-8.
However, when opening them with Notepad in Windows, the characters are displayed correctly regardlessly (which threw me off).

Turns out that Windows' Notepad doesn't always prompt users to change the encoding format, even if the text contains unicode characters.
As a result, the saved files will incorrectly be labeled as ANSI files, and decoded accordingly by text editors in Android, I guess.

In Windows however, all of the editors that I've tried seems to display the characters correctly, regardless of their faulty ANSI labels.
Is this more a question of font types?
